daffodil-lib/src/main/scala/org/apache/daffodil/lib/util/DecimalUtils.scala

```diff
@@ -417,6 +436,25 @@ object DecimalUtils {
       throw new NumberFormatException("Invalid zoned digit: " + digit)
   }
 
+  /**
+   * Does not encode to the EBCDIC charset, but converts negative digits
+   * to their EBCDIC Zoned overpunched representation where negative 0 to 9
+   * are mapped to "}JKLMNOPQR".
+   * @param digit - the digit, as a char '0' to '9'
+   * @param positive - true if positive, false if negative
+   * @return the character needed to represent this character as an overpunched sign digit.
```

Comment:
From the spec:

> In EBCDIC-based encodings, code points 0xC0 to 0xC9 or 0xF0 to 0xF9 represent a positive sign and digits 0 to 9 (these byte ranges correspond typically to characters '{ABCDEFGHI' or '0123456789'), and code points 0xD0 to 0xD9 or 0xB0 to 0xB9 represent a negative sign and digits 0 to 9 (these byte ranges correspond typically to characters '}JKLMNOPQR' or  '^£¥·©§¶¼½¾ ' ). On parsing both ranges are accepted. On unparsing the range 0xC0 to 0xC9 are produced for positive signs and the range 0xD0 to 0xD9 are produced for negative signs.

This makes it sound like on unparsing the positive 0-9 digits should be unparsed as `{ABCDFGHI` instead of 0-9?

Reply:
You are correct. Fixed.